docs/getting-started/rulesets.md

```diff
@@ -18,6 +18,7 @@ rules:
       function: truthy
 ```
 
+Spectral has a built-in set of functions which you can reference in your rules. This example uses the `truthy` to create a rule that checks that the value is true.
 The example above adds a single rule that checks that tags objects have a description property defined.
```

Comment:
"The example above" sounds a little funny now as we already say "This example". Can you merge the two sentences to make it less redundant?

Reply:
Yeah, you are right. It does sounds awkward. Will merge